src/test/unit/micm/micm_c_api.cpp

```diff
@@ -179,8 +192,8 @@ TEST_F(MicmCApiTest, GetUserDefinedReactionRatesOrdering)
   DeleteMappings(reaction_rates_ordering, array_size);
 }
 
-// Test case for solving the MICM instance
-TEST_F(MicmCApiTest, SolveMicmInstance)
+// Test case for solving system using vector-ordered Rosenbrock solver
+TEST_F(MicmCApiTest, SolveUsingVectorOrderedRosenbrock)
```

Comment:
should we also have a test with multiple grid cells?

Reply:
Yes, I think so but this PR doesn't support multiple grid cells yet. Do you think your new task of supporting multiple grid cell can add those test?

Reply:
yes, I agree